content/20-exploring-the-cluster/21-exploring-the-cluster/index.en.md

```diff
@@ -26,93 +25,96 @@ At the top of the page, select the search bar and search for **`"CloudFormation"
+> The workshop environment uses self-signed certificates for learning purposes. In a production environment, always use certificates issued from a trusted Certificate Authority. [AWS Certificate Manager (ACM)](https://aws.amazon.com/certificate-manager/) makes it easy to provision, manage, deploy, and renew SSL/TLS certificates.
 
 ![aws-console-cf-mgmt](/static/images/content/21-aws-cf-mgmt.png)
 ![aws-console-cf-mgmt](/static/images/content/21-aws-cf-mgmt-detail.png)
 
-
 ## Rancher Multi-Cluster Manager
 
 ### Step 1:
 
-Once the Rancher Multi-Cluster Manager opens, we will need to complete the bootstrapping steps.
+Next, we need to complete the bootstrap process in the Rancher Multi-Cluster Manager.
 
-First let's enter the bootstrap password (that was set during installation) of **`"Pa22word"`** and **`Log in with Local User`**.
+First, enter the bootstrap password that was set during installation: **`"Pa22word"`**. Select **`Log in with Local User`**.
 
 ![rancher-bootstrapping-password](/static/images/content/21-rancher-bootstrap-pw.png)
 ![rancher-bootstrapping-password](/static/images/content/21-rancher-bootstrap-pw-detail.png)
 
 ### Step 2:
 
-Now let's verify the few items on the setup page and click **`Continue`**!
+Next, verify a few items on the setup page and click **`Continue`**!
 
-* **`Server URL`** (should not need any changes)
-* **`[ ] Allow collection of anonymous statistics to help us improve rancher`**
-* **`[X] By checking the box, you accept the End User License Agreement & Terms & Conditions`**
+- **`Server URL`** (should not need to be changed)
+- **`[ ] Allow collection of anonymous statistics to help us improve Rancher`**
+- **`[X] By checking the box, you accept the End User License Agreement & Terms & Conditions`**
 
 ![rancher-bootstrapping-setup](/static/images/content/21-rancher-bootstrap-setup.png)
 ![rancher-bootstrapping-setup](/static/images/content/21-rancher-bootstrap-setup-detail.png)
 
 ### Step 3:
 
-Welcome to the Rancher Multi-Cluster Manager!! The starting page in the Rancher Multi-Cluster Manager provides an overview of the cluster managed by this MCM cluster.
+Welcome to the Rancher Multi-Cluster Manager! The starting page in the Rancher Multi-Cluster Manager provides an overview of the cluster(s) managed by this MCM cluster.
 
-Under **`Clusters`**, you should see a cluster named **`local`**. This cluster is your **`management cluster`**. As we move through the workshop, we will be creating and importing additional clusters into the Rancher Manager, known as **`downstream cluster(s)`**.
+Under **`Clusters`**, you should see a cluster named **`local`**. This cluster is your **`management cluster`**. As we move through the workshop, we will be creating and importing additional clusters with the Rancher Manager. These are known as **`downstream cluster(s)`**.
```

Comment:
Change to "Under **`Clusters`**, you should see a cluster named **`local`**. This cluster is your **`management cluster`**. As we move through the workshop, we will be creating and importing additional clusters with the Rancher Manager. These are known as **`downstream cluster(s)`** and should be use for workloads and applications."?